modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/WorkspaceExtension.java

```diff
@@ -365,6 +375,53 @@ private String _decodeBundleUrl(ProductInfo productInfo) {
 		}
 	}
 
+	private String _getBundleChecksumMD5() {
+		String bundleUrl = getBundleUrl();
+
+		if (Objects.isNull(bundleUrl) || bundleUrl.isEmpty()) {
+			return null;
+		}
+
+		try {
+			DownloadCommand downloadCommand = new DownloadCommand();
+
+			downloadCommand.setCacheDir(_workspaceCacheDir);
+			downloadCommand.setPassword(null);
+			downloadCommand.setToken(false);
+			downloadCommand.setUrl(new URL(bundleUrl + ".MD5"));
+			downloadCommand.setUserName(null);
+			downloadCommand.setQuiet(true);
+
+			downloadCommand.execute();
+
+			Path md5Path = downloadCommand.getDownloadPath();
+
+			String md5Content = FileUtil.read(md5Path.toFile());
+
+			int lastSlash = bundleUrl.lastIndexOf('/');
+
+			if (lastSlash == -1) {
+				return null;
+			}
+
+			if (Objects.isNull(md5Content)) {
+				return null;
+			}
+
+			int md5Length = md5Content.indexOf(' ');
+
+			if (md5Length == -1) {
+				return null;
+			}
+
+			return md5Content.substring(0, md5Length);
+		}
+		catch (Exception exception) {
+			throw new GradleException(
```

Comment:
we shouldn't throw a gradle exception if md5 does exist, we should just return silently and continue.